uint8 target_fiducial_id = 0
---
bool is_sucess